### ISSUE TITLE
python bindigns

### ISSUE THREAD

**Issue by pandacriss** [trishume/eyeLike#21]
could you add a way to call the find eyes function from python? tat would be very helpfull

**Comment by trishume** [comment]
I'm not working on this project anymore. Someone could write this in a fork though and link it here.

**Comment by progmars** [comment]
In case if somebody is looking for the same:
I found it already done here: https://github.com/abhisuri97/pyeLike

I have tried some other similar projects based on the same idea and paper:
https://github.com/noelledavis/gaze_tracking
https://github.com/jonnedtc/PupilDetector
however, they are slow and very often not very accurate.

So, eyeLike (and pyeLike) are the most accurate and fast solutions. I fed a bunch of test images and there were some failures, but I can deal with them applying `face_recognition` library https://github.com/ageitgey/face_recognition (wrapper around dlib) to extract eye region and then to reduce it even more to leave only inner area of the eye, and also applying `img = cv2.equalizeHist(img)` (but do it only after extracting the inner region of the eye, otherwise it can make eyeLike work worse).
